docs/nodes/run/FAQ.md

```diff
@@ -23,3 +23,15 @@ is running and not terminated.
 There may be a number of reasons for this issue, but it is likely that the node is not 
 connected properly to other validators, which is usually caused by networking 
 misconfiguration (wrong public IP, closed p2p port 9651).
+
+### Error: `RPCChainVM protocol version mismatch between AvalancheGo and Virtual Machine plugin`
+
+This error occurs when the RPCChainVM protocol version used by Subnet-EVM is incompatible with
+the protocol version of AvalancheGo. Make sure both are using compatible versions. You can 
+follow the table provided below to check the Subnet-EVM versions compatible with each AvalancheGo release:
+
+| RPCChainVM | AvalancheGo | Subnet-EVM | Precompile-EVM | HyperSDK |
+| :--------: | :-------: | :-------: | :-------: | :-------: |
+| 26 | v1.10.1-v1.10.4 | v0.5.1-v0.5.2 | v0.1.0-v0.1.1 | v0.0.6-v0.0.9 |
+| 27 | v1.10.5-v1.10.8 | v0.5.3 | v0.1.2 | v0.0.10-v0.0.12 |
+| **28** | v1.10.9-**v1.10.11(latest)** | v0.5.4-**v0.5.6 (latest)** | v0.1.3-**v0.1.4 (latest)** | v0.0.13-**v0.0.14 (latest)** |
```

Comment:
I hope there is automation tied to the release process to manage this. Having this matrix go stale is arguably worse than not having one

Reply:
For now, it's updated manually. I'll look into what automation we can implement to avoid outdated info.

Reply:
We can provide a link to the version compatibility instead of having a table here, should be easier to maintain that way.